ui/packages/image/src/Image.svelte

```diff
@@ -146,6 +147,17 @@
 			static_image = undefined;
 		}
 	}
+
+	$: {
+		if (cropper) {
+			if (value) {
+				cropper.image = value;
+				cropper.create();
+			} else {
+				cropper.destroy();
+			}
+		}
+	}
```

Comment:
@pngwn not sure if this is the cleanest way to check for value change; then update the cropper.

Reply:
Looks fine, not sure if there is a cleaner way to do it without confusing ternaries really.